core/src/main/java/hudson/cli/handlers/GenericItemOptionHandler.java

```diff
@@ -57,12 +57,12 @@ protected GenericItemOptionHandler(CmdLineParser parser, OptionDef option, Sette
     protected abstract Class<T> type();
 
     @Override public int parseArguments(Parameters params) throws CmdLineException {
-        final Jenkins j = Jenkins.getInstance();
+        final Jenkins j = Jenkins.get();
         final String src = params.getParameter(0);
         T s = j.getItemByFullName(src, type());
         if (s == null) {
             final Authentication who = Jenkins.getAuthentication();
-            try (ACLContext _ = ACL.as(ACL.SYSTEM)) {
+            try (ACLContext aclContext = ACL.as(ACL.SYSTEM)) {
```

Comment:
Good point since the `_` becomes a reserved keyword.

Reply:
was fixed in another PR